x-pack/test/functional/apps/dashboard/_async_dashboard.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getService, getPageObjects }: FtrProviderContext) {
+  const retry = getService('retry');
+  const kibanaServer = getService('kibanaServer');
+  const log = getService('log');
+  const pieChart = getService('pieChart');
+  const find = getService('find');
+  const renderable = getService('renderable');
+  const dashboardExpect = getService('dashboardExpect');
+  const appMenu = getService('appsMenu');
+  const PageObjects = getPageObjects([
+    'common',
+    'header',
+    'home',
+    'discover',
+    'dashboard',
+    'timePicker',
+  ]);
+
+  describe('sample data dashboard', function describeIndexTests() {
+    this.tags('smoke');
```

Comment:
this tag is no longer supported. We do have `includeFirefox`, but we should avoid more Firefox tests on CI to keep it stable